x/clp/types/params.go

```diff
@@ -15,7 +15,10 @@ const (
 
 // Parameter store keys
 var (
-	KeyMinCreatePoolThreshold = []byte("MinCreatePoolThreshold")
+	KeyMinCreatePoolThreshold       = []byte("MinCreatePoolThreshold")
+	KeyLiquidityRemovalLockPeriod   = []byte("LiquidityRemovalLockPeriod")
+	KeyLiquidityRemovalCancelPeriod = []byte("LiquidityRemovalCancelPeriod")
+	KeyRewardPeriods                = []byte("RewardPeriods")
```

Comment:
Does the whole list need to be passed, every time we want to append a rewards period?

Reply:
Yes. Are you thinking of another way it could be done?